src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -728,18 +729,27 @@ public object ToScriptSource(int executionContextId, object executionContextAuxD
         }
     }
 
+    internal class CustomResolver : DefaultAssemblyResolver
+    {
+
+        public void Register(AssemblyDefinition assembly)
```

Comment:
Nit: feel free to ignore

```suggestion
        public void RegisterAssembly(AssemblyDefinition assembly)
```